jars/shaderpacks/default/shaders/gui.vert

```diff
@@ -4,11 +4,11 @@ layout(location = 0) in vec3 position_in;
 layout(location = 1) in vec2 uv_in;
 layout(location = 2) in vec3 color_in;
 
-layout(shared, std140) uniform gui_uniforms {
-    float viewWidth;
-    float viewHeight;
-    mat4 gbufferModelView;
+layout(binding = 0,std140) uniform gui_uniforms {
```

Comment:
We shouldn't need to specify the binding here, Nova should pick it up at runtime. Plus, if you do specify the binding, you'll have to put that on the UBO every time it's used in a shader